hpOneView/resources/servers/enclosure_groups.py

```diff
@@ -46,6 +46,7 @@ class EnclosureGroups(object):
     def __init__(self, con):
         self._connection = con
         self._client = ResourceClient(con, self.URI)
+        self.__default_values = {"type": "EnclosureGroupV200"}
```

Comment:
This is a python constructor, right?
If so, I don't see a problem in setting the minimum supported type at this point.
I would actually recommend it :+1: